<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScorePanel</class>
 <widget class="QWidget" name="ScorePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#ffffff
</string>
  </property>
  <widget class="QPushButton" name="anotherGame">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>580</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Inter&quot;, &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, sans-serif;
font-size: 18px;
font-weight: 500;
color: #2D3748;
background-color: #FFFFFF;
border-radius: 10px;
border: 2px solid #E2E8F0;
</string>
   </property>
   <property name="text">
    <string>再来一局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backToMainMenu">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>-1</pointsize>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Inter&quot;, &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, sans-serif;
font-size: 18px;
font-weight: 500;
color: #2D3748;
background-color: #FFFFFF;
border-radius: 10px;
border: 2px solid #E2E8F0;
</string>
   </property>
   <property name="text">
    <string>回到菜单</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="plainText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>651</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 234)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="winOrLoseText">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>571</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Inter&quot;, &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, sans-serif;
font-size: 28px;
font-weight: 500;
color: #2D3748;
background-color: #FFFFFF;
border-radius: 10px;
border: 2px solid #E2E8F0; 
text-align: center;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="rankText">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Inter&quot;, &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, sans-serif;
font-size: 28px;
font-weight: 500;
color: #2D3748;
background-color: #FFFFFF;
border-radius: 10px;
border: 2px solid #E2E8F0; 
text-align: center;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="surviveTimeText">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Inter&quot;, &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, sans-serif;
font-size: 28px;
font-weight: 500;
color: #2D3748;
background-color: #FFFFFF;
border-radius: 10px;
border: 2px solid #E2E8F0; 
text-align: center;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
